mentat/code_edit_feedback.py

```diff
@@ -22,6 +22,7 @@ async def get_user_feedback_on_edits(
     user_response = user_response_message.data
 
     need_user_request = True
+    total_changes = sum([len(file_edit.replacements) or 1 for file_edit in file_edits])
```

Comment:
I'm not sure how much sense this makes; some formats don't really have set replacements (like unified diff format), not to mention this doesn't take into account creation/deletion/renaming changes. I think we should just keep the regular Changes applied. text and only print it if we do end up applying an edit.